packages/create-sitecore-jss/src/templates/angular/scripts/install-pre-commit-hook.js

```diff
@@ -0,0 +1,28 @@
+﻿const fs = require('fs');
+
+const installHooks = () => {
+  // data to be written to the file
+  const data = `#!/bin/sh
+
+#
+
+# pre-commit hook that runs our linter before every commit
+
+#
+
+echo "Running lint check..."
+yarn pre-commit`;
```

Comment:
Switch to "precommit" name, and we should use npm (our current assumed package manager)
```suggestion
npm run precommit`;
```